Source/WebCore/page/FrameView.cpp

```diff
@@ -2451,6 +2457,29 @@ void FrameView::scrollToFocusedElementInternal()
     FrameView::scrollRectToVisible(absoluteBounds, *renderer, insideFixed, { m_selectionRevealModeForFocusedElement, ScrollAlignment::alignCenterIfNeeded, ScrollAlignment::alignCenterIfNeeded, ShouldAllowCrossOriginScrolling::No });
 }
 
+void FrameView::textFragmentIndicatorTimerFired()
+{
+    Ref protectedThis { *this };
+    
+    ASSERT(frame().document());
+    auto& document = *frame().document();
+    
+    if (!m_textFragmentIndicatorTimerRange)
+        return;
+    
+    auto textIndicator = TextIndicator::createWithRange(m_textFragmentIndicatorTimerRange.value(), { TextIndicatorOption::DoNotClipToVisibleRect }, WebCore::TextIndicatorPresentationTransition::Bounce);
```

Comment:
Do you need to do anything to make sure the range is still valid? Or maybe TextIndicator does it for you?